src/style.css

```diff
@@ -1,3 +1,79 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap';
+
 body {
   margin: 0;
+  background-color: #e1e3e1;
+  font-family: Roboto, system-ui;
```

Comment:
Consider adding a fallback font-family in case 'Roboto' is not available. For instance, you could use a generic font family like 'sans-serif' as a backup.